README.md

```diff
@@ -19,7 +19,7 @@ Follow the README instructions in the directories to try it out:
 
 [`equinix-metal-aka-packet`](equinix-metal-aka-packet)
 
-[`hetzner`](hetzner)
+[`flatcar-terraform-hetzner`](flatcar-terraform-hetzner)
```

Comment:
Why the rename? I don't think a terraform module needs terraform in its name and it's duplicating the repo name.

Reply:
or let's rename all terraform modules if we want to have it this way

Reply:
SEO: This way the string "flatcar-terraform-hetzner" will be in the URL, which bumps google ranking of the Hetzner part of the repo.